### ISSUE TITLE
homing problem

### ISSUE THREAD

**Issue by alifadl09** [MarlinFirmware/Marlin#4367]
I was using an old marlin firmware with auto bed leveling enabled and every think worked great, but after i updated marlin after it runs G29 the nozzle raise then start printing from the air.
I use proximity sensor for auto bed leveling, my start Gcode command is:
G28
G29
G92 Z0.2
it turns out that i forgot to set (z min end stop) to true ,so i set it true but another problem appears ,when i hit z homing button it home to the opposite direction(not until it hit end stop) and auto leveling also .

[marlin_with_autobed_leveling.zip](https://github.com/MarlinFirmware/Marlin/files/376514/marlin_with_autobed_leveling.zip)

﻿

**Comment by brainscan** [comment]
Your probe offset is positive but should be negative if using proximity sensor. Also why the  G92 Z0.2? I think that was a work around on old Marlin that's no longer needed.

Sent from my iPhone

On 21 Jul 2016, at 17:45, alifadl09 <notifications@github.com<mailto:notifications@github.com>> wrote:

I was using an old marlin firmware with auto bed leveling enabled and every think worked great, but after i updated marlin after it runs G29 the nozzle raise then start printing from the air.
I use proximity sensor for auto bed leveling, my start Gcode command is:
G28
G29
G92 Z0.2
it turns out that i forgot to set (z min end stop) to true ,so i set it true but another problem appears ,when i hit z homing button it home to the opposite direction(not until it hit end stop) and auto leveling also .

marlin_with_autobed_leveling.ziphttps://github.com/MarlinFirmware/Marlin/files/376514/marlin_with_autobed_leveling.zip

?

##

**Comment by jbrazio** [comment]
Possible duplicate of #4325
